velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -432,6 +446,34 @@ AggregationFuzzerBase::computeReferenceResults(
       std::nullopt, ReferenceQueryErrorCode::kReferenceQueryUnsupported);
 }
 
+std::pair<
+    std::optional<std::vector<RowVectorPtr>>,
+    AggregationFuzzerBase::ReferenceQueryErrorCode>
+AggregationFuzzerBase::computeReferenceResultsAsVector(
+    const core::PlanNodePtr& plan,
+    const std::vector<RowVectorPtr>& input) {
+  VELOX_CHECK(referenceQueryRunner_->supportsVeloxVectorResults());
+
+  if (auto sql = referenceQueryRunner_->toSql(plan)) {
+    try {
+      return std::make_pair(
+          referenceQueryRunner_->executeVector(
+              sql.value(), input, plan->outputType()),
+          ReferenceQueryErrorCode::kSuccess);
+    } catch (std::exception& e) {
```

Comment:
`const std::exception&` ?

Reply:
Doesnt look like we use it, I can just remove it .